src/collectd.conf.in

```diff
@@ -357,6 +357,11 @@
 #  </View>
 #</Plugin>
 
+#LoadPlugin btrfs
```

Comment:
Please put the `LoadPlugin` line with the other `LoadPlugin` lines (they're sorted alphabetically). Also adope the conditional format that they use.